schematics/nav/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.html

```diff
@@ -0,0 +1,18 @@
+<mat-drawer-container>
+  <mat-drawer mode="side" #drawer [opened]="!isHandset">
```

Comment:
Should the `mode` change based on the screen size? I.e., when it gets large enough it goes to `over`.